packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -22,82 +19,78 @@ import { ListActions as DefaultActions } from './ListActions';
 import { Empty } from './Empty';
 import { ListProps } from '../types';
 
+const defaultActions = <DefaultActions />;
+const defaultBulkActionButtons = <BulkDeleteButton />;
+const defaultPagination = <DefaultPagination />;
+const defaultEmpty = <Empty />;
+const DefaultComponent = Card;
+
 export const ListView = (props: ListViewProps) => {
     const {
-        actions,
+        actions = defaultActions,
         aside,
         filters,
-        bulkActionButtons,
-        pagination,
+        bulkActionButtons = defaultBulkActionButtons,
+        pagination = defaultPagination,
         children,
         className,
-        component: Content,
-        exporter = defaultExporter,
+        component: Content = DefaultComponent,
         title,
-        empty,
+        empty = defaultEmpty,
         ...rest
     } = props;
-    const controllerProps = getListControllerProps(props); // deprecated, to be removed in v4
-    const listContext = useListContext(props);
-
     const {
         defaultTitle,
         total,
-        loaded,
-        loading,
+        isLoading,
         filterValues,
         selectedIds,
-    } = listContext;
-    const version = useVersion();
+    } = useListContext(props);
 
     const renderList = () => (
         <>
             {(filters || actions) && (
-                <ListToolbar
-                    filters={filters}
-                    {...controllerProps} // deprecated, use ListContext instead, to be removed in v4
-                    actions={actions}
-                    exporter={exporter} // deprecated, use ListContext instead, to be removed in v4
-                />
+                <ListToolbar filters={filters} actions={actions} />
             )}
             <div className={ListClasses.main}>
                 <Content
                     className={classnames(ListClasses.content, {
                         [ListClasses.bulkActionsDisplayed]:
                             selectedIds.length > 0,
                     })}
-                    key={version}
                 >
                     {bulkActionButtons !== false && bulkActionButtons && (
-                        <BulkActionsToolbar {...controllerProps}>
+                        <BulkActionsToolbar>
                             {bulkActionButtons}
                         </BulkActionsToolbar>
                     )}
                     {children &&
                         // @ts-ignore-line
                         cloneElement(Children.only(children), {
-                            ...controllerProps, // deprecated, use ListContext instead, to be removed in v4
                             hasBulkActions: bulkActionButtons !== false,
                         })}
-                    {pagination && cloneElement(pagination, listContext)}
+                    {pagination !== false && cloneElement(pagination)}
```

Comment:
We don't need to clone it anymore, right?

Reply:
Right